src/main/webapp/js/feed-loader.js

```diff
@@ -0,0 +1,50 @@
+// Fetch messages and add them to the page.
+function fetchMessages() {
+    const url = '/feed';
+    fetch(url).then((response) => {
+        return response.json();
+    }).then((messages) => {
+        const messageContainer = document.getElementById('message-container');
+        if(messages.length == 0) {
+            messageContainer.innerHTML = '<p>There are no posts yet.</p>';
+        }
+        else {
+            messageContainer.innerHTML = '';
+        }
+        messages.forEach((message) => {
+            const messageDiv = buildMessageDiv(message);
+            messageContainer.appendChild(messageDiv);
+        });
+    });
+}
+
+function buildMessageDiv(message) {
+    const usernameDiv = document.createElement('div');
+    usernameDiv.classList.add('left-align');
+    usernameDiv.appendChild(document.createTextNode(message.user));
+
+    const timeDiv = document.createElement('div');
+    timeDiv.classList.add('right-align');
+    timeDiv.appendChild(document.createTextNodeNode(new Date(message.timestamp)));
```

Comment:
Ah this is what was causing the error. It should be createTextNode.
```suggestion
    timeDiv.appendChild(document.createTextNode(new Date(message.timestamp)));
```